src/main/java/ru/geekbrains/boot/controllers/ProductController.java

```diff
@@ -0,0 +1,51 @@
+package ru.geekbrains.boot.controllers;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+import ru.geekbrains.boot.model.Product;
+import ru.geekbrains.boot.services.ProductService;
+
+import java.util.Optional;
+
+@Controller
+@RequestMapping("/products")
+@RequiredArgsConstructor
+public class ProductController {
+    private ProductService productService;
+
+    public void ProductsController(ProductService productService) {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.